app/models/diet.rb

```diff
@@ -1,2 +1,10 @@
 class Diet < ApplicationRecord
+  has_and_belongs_to_many :users,
+                          class_name: 'User' ,
```

Comment:
Idk if it's needed to call class_name 'User' when it's :users who habtm

Reply:
true that :)